eo-runtime/src/main/java/EOorg/EOeolang/EOsys/EOunix.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2024 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+/*
+ * @checkstyle PackageNameCheck (4 lines)
+ */
+package EOorg.EOeolang.EOsys;
+
+import org.eolang.AtVoid;
+import org.eolang.Atom;
+import org.eolang.Data;
+import org.eolang.Dataized;
+import org.eolang.PhDefault;
+import org.eolang.Phi;
+
+/**
+ * CALL.
+ *
+ * @since 0.39
+ * @checkstyle TypeNameCheck (100 lines)
+ */
+public final class EOunix extends PhDefault implements Atom {
+
+    /**
+     * Ctor.
+     */
+    public EOunix() {
+        this.add("name", new AtVoid("name"));
+        this.add("args", new AtVoid("args"));
+    }
+
+    @Override
+    public Phi lambda() throws Exception {
+        final Phi name = this.take("name");
+        final Phi[] args = this.collectArgs();
+        return new Data.ToPhi(
+            new DispatchedUnixSyscall(
+                new Dataized(name).take(String.class)
+            ).call(args)
+        );
+    }
+
+    private Phi[] collectArgs() {
+        final Phi args = this.take("args");
+        final Phi retriever = args.take("at");
```

Comment:
@c71n93 maybe it would be faster to use `tuple.head` and `tuple.tail` instead of `tuple.at`.